mne_bids/utils.py

```diff
@@ -363,6 +363,53 @@ def _age_on_date(bday, exp_date):
     return exp_date.year - bday.year - 1
 
 
+def _bday_on_age(age, exp_date):
+    """Calculate birthday from age and experiment date.
+
+    Parameters
+    ----------
+    age : int | float
+        The age of the participant.
+    exp_date : instance of datetime.datetime
+        The date the experiment was performed on.
+
+    Returns
+    -------
+    birthday : tuple of int
+        The (year, month, day) birthday of the subject. If age
+        is fractional age, then the month and day are also
+        calculated. Else the month and day default to (1, 1).
```

Comment:
My suggestion would be to leave the birthday field blank if it's not in the `raw` file. Guessing or setting to a random value will only lead to bad things.

Reply:
I tend to agree unless you want the round trip with metadata. But yes the proposal here seems really error prone to me

Reply:
Is it sufficient then to just set to `(computed_year, 1, 1)`? Else, I think it's impossible to infer birthday to a sufficient degree. 

If so, then I can leave the birthday field out. Since mne-python doesn't have an age field, then I think we won't be able to roundtrip this?